applications/hotornot/src/hon_rater.erl

```diff
@@ -12,49 +12,55 @@
 
 -include("hotornot.hrl").
 
--spec init() -> ok.
-init() -> kapps_maintenance:refresh(?KZ_RATES_DB).
-
--spec handle_req(kz_json:object(), kz_proplist()) -> 'ok'.
-handle_req(JObj, _Props) ->
-    'true' = kapi_rate:req_v(JObj),
-    _ = kz_util:put_callid(JObj),
+-spec init() -> 'ok'.
+init() ->
+    _ = [kapps_maintenance:refresh(kzd_ratedeck:format_ratedeck_db(Ratedeck))
+         || Ratedeck <- hotornot_config:ratedecks()
+        ],
+    'ok'.
```

Comment:
Seems like its 6 one way, half a dozen the other. Not sure one is better than the other in this particular case.